src/lib/bootloader/udev_mapping.rb

```diff
@@ -105,48 +94,28 @@ def kernel_to_udev(dev)
       result
     end
 
-    # picks udev name according to strategy
+    # Udev name for the device
+    #
+    # The selected udev name depends on the mount by option. In case of an unmounted device,
+    # storage-ng has logic to discover the preferred mount by option.
+    #
     # @see #to_mountby_device
+    #
+    # @return [String]
     def udev_name_for(device)
-      mount_by_udev(device) ||
-        udev_by_label(device) ||
-        udev_by_uuid(device) ||
-        udev_by_id(device) ||
-        udev_by_path(device) ||
-        device.name
+      mount_by_udev(device) || device.name
```

Comment:
NP: I am not sure if it is worth to have it in two methods as logic now is quite small.

Reply:
Ok